### PR TITLE
extended header with logo and made it somewhat responsive

### PR DESCRIPTION
**Pull request by kimamov**

### CODE REVIEW

**Comment by kimamov** [comment]
looks like typescript is somewhat of a bully with static files images etc. and the linter does not allow ignoring the line

File: package.json [hunk 1]
```diff
@@ -1,58 +1,161 @@
...
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/icons": "^4.11.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0"
-  }
-}
+{
+
```
Comment (lopis) [inline package.json]: 😱  what happened here?

**Comment by kimamov** [comment]
looks like resolving conflicts in yarn.lock is kinda messy I will just do it in my IDE tomorrow

File: src/pages/AddJobPage.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Container from '@material-ui/core/Container';
+import React, { ReactElement } from 'react';
+import PageLayout from '../layouts/PageLayout';
+
+function AddJobPage(): ReactElement {
+  return (
+    <PageLayout>
+      <Container>to be implemented</Container>
```
Comment (lopis) [inline src/pages/AddJobPage.tsx:8]: I only meant this to be a link to the form, but actually this page might be useful.

File: src/customTypes.d.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+declare module '*.jpg';
+declare module '*.png';
+declare module '*.jpeg';
+declare module '*.gif';
```
Comment (lopis) [inline src/customTypes.d.ts]:
```suggestion
declare module '*.gif';

```
Missing new line

**Review by lopis — CHANGES_REQUESTED** [review]

**Review by lopis — APPROVED** [review]

**Pull request merged**